apps/storybook-react-native/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,413 @@
+import {
+  AvatarAccount,
+  AvatarAccountVariant,
+  AvatarBaseShape,
+  AvatarToken,
+  AvatarTokenSize,
+  BadgeCount,
+  BadgeWrapper,
+  BadgeNetwork,
+  Button,
+  ButtonBase,
+  ButtonIcon,
+  ButtonSize,
+  ButtonVariant,
+  FontWeight,
+  Icon,
+  IconName,
+  IconColor,
+  Text,
+  TextButton,
+  TextColor,
+  TextVariant,
+  BadgeWrapperPosition,
+  BadgeStatus,
+  BadgeStatusStatus,
+  IconSize,
+} from '@metamask/design-system-react-native';
+import { useTailwind } from '@metamask/design-system-twrnc-preset';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { View, ScrollView, Pressable } from 'react-native';
+
+const meta: Meta = {
+  title: 'Examples/Wallet Home',
+  component: () => null,
+};
+
+export default meta;
+type Story = StoryObj;
+
+const WalletHome: React.FC = () => {
+  const tw = useTailwind();
+
+  return (
+    <ScrollView style={tw`bg-background-default flex-1`}>
+      {/* Container */}
+      <View style={tw`bg-background-default w-full py-4`}>
```

Comment:
Need to allow for `bg-default` and other shorthand design token classnames similar to react